keras/layers/merge.py

```diff
@@ -187,6 +187,25 @@ class Add(_Merge):
     It takes as input a list of tensors,
     all of the same shape, and returns
     a single tensor (also of the same shape).
+
+    # Examples
+
+    ```python
+        from keras.layers.merge import Add  # a layer
+        from keras.layers.merge import add  # a functional interface
+        from keras.layers import Input, Dense
+        from keras.models import Model
```

Comment:
You can do:
`from keras import layers`
`layers.add`

Or:
`keras.layers.add`